redfish-core/lib/assembly.hpp

```diff
@@ -209,6 +209,36 @@ void getAssemblyLocationCode(
     });
 }
 
+void getAssemblyPresence(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+                         const auto& serviceName, const auto& assembly,
+                         const auto& assemblyIndex)
+{
+    sdbusplus::asio::getProperty<bool>(
+        *crow::connections::systemBus, serviceName, assembly,
+        "xyz.openbmc_project.Inventory.Item", "Present",
+        [asyncResp, assemblyIndex](const boost::system::error_code ec,
+                                   const std::variant<bool>& property) {
```

Comment:
Can this be simplified as
```
      const s bool value) {
```

And remove L228-L234?

Reply:
done